ext/pdo_mysql/tests/pdo_mysql___construct_uri.phpt

```diff
@@ -73,5 +71,5 @@ MySQLPDOTest::skip();
     print "done!";
 ?>
 --EXPECTF--
-[003] URI=uri:file://%spdomuri.tst, DSN=mysql%sdbname=%s, File=%spdomuri.tst (%d bytes, 'mysql%sdbname=letshopeinvalid%s'), chr(0) test, EXPECTED ERROR
+[003] URI=uri:file://%spdomuri.tst, DSN=mysql:%sdbname=%s, File=%spdomuri.tst (%d bytes, 'mysql:%sdbname=letshopeinvalid%snonsense'), EXPECTED ERROR
```

Comment:
Why did the output change?

Reply:
`, chr(0) test` is not needed here as the line/echo is "identified" by `[003]`, other additions here are to improve the expectation